### ISSUE TITLE
Lithostratigraphy isn't hierarchical

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2479]
When we start thinking about moving to the locality attribute model we need to think about how to handle non-hierarchical terms in lithostratigraphy.

Here is an example - Greenhorn Limestone Formation is part of two different groups depending upon location. See [Macrostrat](https://macrostrat.org/sift/#/strat_name/3617).

Greenhorn Limestone Formation of Benton Group (where it is now in the code table)
Greenhorn Limestone Formation of Colorado Group

Currently we can't handle this. One idea has been to add something to the formation name so that we have unique names. So maybe

Greenhorn Limestone Formation (Benton Group)
Greenhorn Limestone Formation (Colorado Group)

We need to figure it out eventually.

**Comment by dustymc** [comment]
Macrostrat is "identification" not "taxonomy" - they are NOT an "authority" they just list what people have called things (maybe using some authoritative terminology).

A formation is a subset of a group - if it's not hierarchical, it's not really a formation and/or group. I think this is again just a case of "identification" instead of "taxonomy" (and I'm sure it'll all be worked out in the next few millennia....).

IOW our model is designed to handle the data as designed, which nobody actually uses because there are too many unknowns. 

That's really only a problem because we're mixing assertions and authorities (because the person who designed the model wasn't aware of the reality...). As I've suggested before, we should just stop. Collector says "Greenhorn Limestone Formation, Benton Group" we make two assertions and that's it. Collector says "Benton Group" or "Greenhorn Limestone Formation" we make one assertion. Remove the implicit part of the model/authority altogether.

I think that's viable because things like macrostrat exists. I can take the data, which can't be confounded with anything else in a simple "here's what {agent} said" model, and get back potential parentage, age, bla bla bla.

Basically I'm suggesting we move the hierarchical component of our model to webservices, which are designed to "think" like geologists seem to do, and not be constrained by designs that don't seem to actually exist at any useful level 'in the wild.'

**Comment by Jegelewicz** [comment]
I think we have resolved this with the above.
